app/javascript/consent/ConsentComponent.js

```diff
@@ -10,8 +10,8 @@ import type { AppState } from '../state/reducers';
 import './ConsentComponent.css';
 
 type Props = {
-  isNewMember: boolean,
-  isRequired: boolean,
+  shouldRenderHidden: boolean,
```

Comment:
Why'd you decide to rename these? `isNewMember` seemed more illustrative of the condition in which the hidden fields should be rendered than `shouldRenderHidden`. Mebbe you could add a comment or smthn.